templates/example_MPH_plots.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>This notebook should be become 3d, with an updated description in the Readme</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>